pyamf/amf0.py

```diff
@@ -234,13 +234,13 @@ def readMixedArray(self):
 
         attrs = self.readObjectAttributes(obj)
 
-        for key, value in attrs.iteritems():
+        for key in attrs.keys():
```

Comment:
looks like I chose wrong while merging this change from the WK version, which hadn't had https://github.com/hydralabs/pyamf/commit/dbe5d4e2aa7ff8527917ec0475bee7216108b56b applied.